liquibase/dbchangelog.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+    <changeSet id="1" author="jossuecito">
+        <!-- Dependencies table -->
+        <createTable tableName="fileurls">
+            <column name="id" type="int">
+                <constraints primaryKey="true" primaryKeyName="pk_fileurls"/>
+            </column>
+            <column name="url" type="varchar(2000)">
```

Comment:
Can we add comments at all in this xml spec? I'd like to see a comment here saying something about how this is the practical upper bound on URL length

Reply:
Yeah we can add comments